docs/docs/CHANGELOG.md

```diff
@@ -53,9 +53,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed mojeekbot user agent regex
 - Added support for running anubis behind a base path (e.g. `/myapp`)
 - Reduce Anubis' paranoia with user cookies ([#365](https://github.com/TecharoHQ/anubis/pull/365))
-- Added support for Opengraph passthrough while using unix sockets
-- The opengraph subsystem now passes the HTTP `HOST` header through to the origin
```

Comment:
I didn't see anything with a filename of `opengraph`, so I think using the title case naming is appropriate.